binding-mri/binding-mri.cpp

```diff
@@ -147,7 +147,24 @@ static void mriBindingInit()
 	_rb_define_module_function(mod, "raw_key_states", mkxpRawKeyStates);
 	_rb_define_module_function(mod, "mouse_in_window", mkxpMouseInWindow);
 
+	/* Load global constants */
+	bool has_debug = false;
+	bool has_test = false;
+	bool has_btest = false;
+	for (int i = 1; i < shState->rtData().argc; i++)
+	{
+		const std::string arg = shState->rtData().argv[i];
+		if ("debug" == arg)
+			has_debug = true;
+		else if ("test" == arg)
+			has_test = true;
+		else if ("btest" == arg)
+			has_btest = true;
+	}
 	rb_gv_set("MKXP", Qtrue);
+	rb_gv_set("DEBUG", has_debug ? Qtrue : Qfalse);
+	rb_gv_set("TEST", has_test ? Qtrue : Qfalse);
+	rb_gv_set("BTEST", has_btest ? Qtrue : Qfalse);
```

Comment:
There is a convenience function for C++ → Ruby bool conversion in `binding-util.h`